### PR TITLE
Add a lambda function to the backend that can receive data

### PR DESCRIPTION
**Pull request by jkzx**
This PR
        - Creates a lambda function that can receive data
        - Logs the received data

### CODE REVIEW

File: cream-services/create.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export function main(event, context, callback) {
+    console.log(event);
```
Comment (MadTribe) [inline cream-services/create.js]: The server needs to return a response or the the client will sit there waiting.  Should be 200 for success

Comment (MadTribe) [inline cream-services/create.js]: I think we should use this PR to validate the data format and parse the data into an object.

File: cream-services/create.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export function main(event, context, callback) {
```
Comment (MadTribe) [inline cream-services/create.js]: Is "main" the recommended name for this function?

File: cream-services/create.js (hunk 1 above)
Comment (jkzx) [reply cream-services/create.js]: Will add a 200 for success

Comment (jkzx) [reply cream-services/create.js]: Added the validate part

File: cream-services/create.js (hunk 2 above)
Comment (jkzx) [reply cream-services/create.js]: Changed to "create".

**Pull request merged**